src/Core/tests/DeviceTests/Stubs/SearchBarStub.cs

```diff
@@ -18,6 +18,8 @@ public partial class SearchBarStub : StubBase, ISearchBar
 
 		public TextAlignment HorizontalTextAlignment { get; set; }
 
+		public TextAlignment VerticalTextAlignment { get; set; }
+
```

Comment:
Could you include a test that verify the new property?

Reply:
Added device tests similar to what we have for HorizontalTextAlignment